aten/src/ATen/PythonModeTLS.cpp

```diff
@@ -0,0 +1,53 @@
+#include <ATen/PythonModeTLS.h>
+
+namespace at { namespace impl {
+
+#if !defined(CAFFE2_IS_XPLAT_BUILD) && !defined(C10_MOBILE)
```

Comment:
Why is this protected by mobile macros?

Reply:
The OSS mobile build fails unless these are here. I didn't dig into the failures, do you think it is worth it?

Reply:
I'm experimenting with the builds now

Reply:
Well, I'm trying to figure out some other mobile related failures that I can't understand.
This feature is expected not to work on mobile? Can we make it an explicit error?

Reply:
The feature definitely doesn't work on mobile because (correct me if I'm wrong) there is no python interpreter associated with mobile so there is no way to actually call into the public APIs for python mode

Reply:
Removed the mobile macros and everything still compiles + tests pass so I'm going to leave it in the current state. If the mobile folks end up wanting to save space by disabling the feature they can go for it in a follow up